materials/job_types.md

```diff
@@ -85,7 +86,14 @@ In this course we will focus on **embarrassingly/naturally/delightfully parallel
 
 ## Array jobs
 
-[Array jobs](https://docs.csc.fi/computing/running/array-jobs/) are another way of taking advantage of Puhti's parallel processing capabilities for embarrassingly parallel tasks. Array jobs are useful when same code is executed many times for different datasets or with different parameters without the need to change the Python code. In GIS context a typical use case would be to run some model on study area split into multiple files where output from one file doesn't have an impact on the result of an other area. 
+[Array jobs](https://docs.csc.fi/computing/running/array-jobs/) are another way of taking advantage of Puhti's parallel processing capabilities for embarrassingly parallel tasks. Array jobs are useful when same code is executed many times for different datasets or with different parameters without the need to change your code. In GIS context, a typical use case would be to run some model on study area split into multiple files where output from one file doesn't have an impact on the result of another area. 
+
+:::{admonition} Maximum job limits
+:class: warning
+
+Submitting an array job of 100 members counts the same as 100 individual jobs from the batch queue system's perspective. The maximum number of jobs that each user can submit per month should be kept below one thousand. 
```

Comment:
```suggestion
Submitting an array job of 100 members counts the same as 100 individual jobs from the batch queue system's perspective. One can submit/run a maximum of 200/400 jobs at the same time. The maximum number of jobs that each user can submit per month should be kept below one thousand. 
```

Reply:
:+1: also probably good to mention explicitly that this applies for Puhti. Limits are different for different systems, e.g. Mahti

Reply:
updated